src/Native/Runtime/startup.cpp

```diff
@@ -203,8 +203,7 @@ bool UninitDLL(HANDLE /*hModDLL*/)
     buffer[len++] = '\r';
```

Comment:
`\r` will look ugly on Unix. They should be stripped from everywhere. (I noticed that there are a few in GC too.)

I believe that Windows CRT just adds them transparently for you - e.g. if you write `\n` to stdout, it will get expanded to `\r` `\n` underneath to follow the Windows conventions. So we should be able to strip them without any ifdefs, etc.